Css.css

```diff
@@ -0,0 +1,99 @@
+body{
+    color: rgb(255, 255, 255);
+    font-family: 'Courier New', Courier, monospace;
+    border-image-width: auto;
+    margin-top: auto;
+    padding: 0%;
+    background-image: url(imgs/bkgdContact.jpg)
+    
+}
+  /* For mobile phones: */
+  @media only screen and (max-width: 360px) {
+    body {
+      background-color: lightblue;
```

Comment:
cannot see background color because you are using a background image,   pick elements to change so that with the different sizes you can visually see differences on the web page

example: have 3 different heading colors based on the max-width

Reply:
I see the updates now!!!